src/search.cpp

```diff
@@ -387,7 +391,46 @@ void MainThread::search() {
       for (Thread* th : Threads)
           if (   th->completedDepth > bestThread->completedDepth
               && th->rootMoves[0].score > bestThread->rootMoves[0].score)
-              bestThread = th;
+              longestPVThread = bestThread = th;
+
+      if (bestThread->rootMoves[0].pv.size() < minPlies) {
+          for (Thread* th : Threads) {
+              // Look for the best thread that meets the minimum move criteria.
```

Comment:
Please wrap comments in a manner friendly to GitHub diff capability (column 80 I think?)